src/list/actions.js

```diff
@@ -36,13 +36,26 @@ export const HIDE_MODAL = Symbol("HIDE_MODAL");
 
 export const SEND_FEEDBACK = Symbol("SEND_FEEDBACK");
 export const OPEN_FAQ = Symbol("OPEN_FAQ");
+export const OPEN_SYNC_PREFS = Symbol("OPEN_SYNC_PREFS");
+export const OPEN_ACCOUNT = Symbol("OPEN_ACCOUNT");
+export const OPEN_CONNECT_DEVICE = Symbol("OPEN_CONNECT_DEVICE");
+
+export const SELECT_TAB_LOGINS = Symbol("SELECT_TAB_LOGINS");
+export const SELECT_TAB_MONITOR = Symbol("SELECT_TAB_MONITOR");
+
+export const GET_PROFILE = Symbol("GET_PROFILE");
+export const GET_PROFILE_STARTING = Symbol("GET_PROFILE_STARTING");
+export const UPDATED_PROFILE = Symbol("UPDATED_PROFILE");
 
 // The action ID is used for debugging to correlate async actions with each
 // other (i.e. FOO_STARTING and FOO_COMPLETED).
 let nextActionId = 0;
 
 const FEEDBACK_URL = "https://qsurvey.mozilla.com/s3/Lockbox-Input?ver=" + version;
 const FAQ_URL = "https://mozilla-lockbox.github.io/lockbox-extension/faqs/";
```

Comment:
One of the intents of #72 was to update the FAQ an Feedback links -- the issue has the correct link for the FAQ, but we don't yet have the correct Feedback link (will have a followup)